src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1351,12 +1351,18 @@ TEST_F(PolicyHandlerTest, OnSnapshotCreated_UrlAdded) {
   EndpointUrls test_data;
   EndpointData data("some_data");
   test_data.push_back(data);
+  AppIdURL next_app_url = std::make_pair("default", "some data");
```

Comment:
@tkireva The literal `"some data"` is repeated and may be exposed as constant. The Policy specific constant `policy::kDefaultID` is defined as "default" and may be used.